pnpm/Dockerfile.prod

```diff
@@ -25,7 +25,10 @@ USER node
 
 # Set working directory to /app
 WORKDIR /app
-RUN corepack use pnpm@9
+# Upgrades pnpm to the latest version globally
+RUN corepack install -g pnpm
```

Comment:
@sneridagh Why do we need a global one and not just the one for the project?

Reply:
@davisagli we can remove this, but I thought that we might want to have the global one updated too (it's pnpm 8 still if 20-slim is used, at least the last time I checked). So we first update it globally, then just install the specified version in `package.json` afterwards.